keyboards/cybergear/macro25/keymaps/via/keymap.c

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2021 CyberGear Marius Kavoliunas
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+// https://codepen.io/mvaneijgen/full/LYEVQXp
+
+#define MAC(kc) (QK_RCTL | QK_RGUI | QK_RALT | (kc))
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        MAC(KC_KP_0), MAC(KC_KP_1), MAC(KC_KP_2), MAC(KC_KP_3), MAC(KC_KP_4),
+        MAC(KC_KP_5), MAC(KC_KP_6), MAC(KC_KP_7), MAC(KC_KP_8), MAC(KC_KP_9)
+    )
+};
```

Comment:
The VIA keymap should have as many layers as used by dynamic keymaps. Default is 4
https://caniusevia.com/docs/configuring_qmk#create-a-keymapc-in-keyboardskeyboardnamekeymapsvia